docs/it-manual/configure-gis-service.md

```diff
@@ -1,51 +1,149 @@
+  - Example value: ["https://gisdata.yourcity.gov/server/rest/services/City_Limits/MapServer/1/query"]
+
+
+ - [ESRI_ADDRESS_SERVICE_AREA_VALIDATION_LABELS](https://github.com/civiform/civiform/blob/fd0aaa002e2ee01d378ca90f236c316641ed0101/server/conf/application.conf#L730)
+is the list of labels that CiviForm admins will see when setting up eligibility
+or visibility conditions.
+ 
+   - Type:  String[]
+   
+   - Example values: ["Seattle"], ["Abc County", "Def County", "Ghi County"]
+
+
+- [ESRI_ADDRESS_SERVICE_AREA_VALIDATION_ATTRIBUTES](https://github.com/civiform/civiform/blob/fd0aaa002e2ee01d378ca90f236c316641ed0101/server/conf/application.conf#L733)
+is the list of attributes that should be checked in the response returned from
+the Esri service URL.
+
+  Note that these attributes are custom for each Esri URL and you'll need to look at what fields are provided for your specific map to know what attribute to use. For example, [Seattle's service area validation service](https://gisdata.seattle.gov/server/rest/services/COS/Seattle_City_Limits/MapServer/1) specifies "CITYNAME" as an attribute.
+  
+  - Type:  String[]
+
+  - Example values: ["CITYNAME"], ["ZIPCODE"]
+
+
+- [ESRI_ADDRESS_SERVICE_AREA_VALIDATION_IDS](https://github.com/civiform/civiform/blob/fd0aaa002e2ee01d378ca90f236c316641ed0101/server/conf/application.conf#L731)
+is the list of values that the attribute should be equal to.
+  
+  - Type:  String[]
+  
+  - Example values:
+       
+    - If `_ATTRIBUTES` is set to ["CITYNAME"], then `_IDS` could be set to ["Seattle"] in order to require addresses be in the city of Seattle.
+    - If `_ATTRIBUTES` is set to ["ZIPCODE"], then `_IDS` could be set to ["28202"] in order to require addresses be in the 28202 zip code.
 
-The number of tries CiviForm will attempt requests to external Esri services.
+The configuration allows you to set up multiple service areas so that different questions can have different eligibility requirements. For example, maybe one program allows anyone in the county to apply while a different program only allows people in a certain zip code to apply. You can set up two service areas:
 
+```
+export ESRI_ADDRESS_SERVICE_AREA_VALIDATION_URLS = ["example-county-url", "neighborhood-url"]
+export ESRI_ADDRESS_SERVICE_AREA_VALIDATION_LABELS = ["Example County", "Example Neighborhood"]
+export ESRI_ADDRESS_SERVICE_AREA_VALIDATION_ATTRIBUTES = ["COUNTY", "ZIPCODE"]
+export ESRI_ADDRESS_SERVICE_AREA_VALIDATION_IDS = ["Example County", "01234"]
+```
+
+One program can set their eligibility condition to use "Example County" and the
+other program can set their eligibility condition to use "Example Neighborhood".
+
+The arrays in each of these four variables should all be the same length. The
+values at index `i` should all correlate with each other. In the example above,
+all values at index `0` are for checking the county service area, and all values
+at index `1` are for checking the neighborhood service area.
```

Comment:
**suggestion, non-blocking**

This feature could be confusing, so it might be helpful to restate it or give another example like:
> Put another way:
> - When checking that an address is in the service area labeled `Example County` in the admin UI, CiviForm will query `example-county-url` and check the response for a field called `COUNTY` with a value of `Example County`.
> - When checking that an address is in the service area labeled `Example Neighborhood` in the admin UI, CiviForm will query `neighborhood-url` and check the response for a field called `ZIPCODE` with a value of `01234`.

I might also make the label different from the ID so the example is clearer.

Reply:
These bullets are great, added. I was hoping the neighborhood example would show how the label and ID would be different, but switched the county example to be state instead, where the label is the full state and the id is the abbreviation.